packages/ai-jsx/src/lib/openai.tsx

```diff
@@ -51,7 +51,10 @@ export const openAiClientContext = LLMx.createContext<OpenAIApi>(
   new OpenAIApi(
     new Configuration({
       apiKey: process.env.OPENAI_API_KEY,
-    })
+    }),
+    undefined,
+    // TODO: Figure out a better way to work around NextJS fetch blocking streaming
+    (globalThis as any)._nextOriginalFetch ?? globalThis.fetch
```

Comment:
FYI @NickHeiner it looks like NextJS's `fetch` patch does something not-streaming friendly. So from peeking at their code I found out how I could get the "real" fetch 😒

Reply:
Hmmm, interesting. Good find.